oxidation/crates/parsec_client_types/src/local_manifest.rs

```diff
@@ -111,6 +183,99 @@ impl From<LocalFileManifest> for LocalFileManifestData {
 
 impl_local_manifest_dump_load!(LocalFileManifest);
 
+impl LocalFileManifest {
```

Comment:
![image](https://user-images.githubusercontent.com/3187637/160101759-d69896a3-e97d-4dc7-8904-36c4894e5a1e.png)

Reply:
There is plenty of methods missing (e.g. `new_placeholder/apply_prevent_sync_pattern/evolve_children_and_mark_updated` etc.) for all the local manifests structs

they have ended up in the python binding code, but there should be implemented here !